package.json

```diff
@@ -6,7 +6,13 @@
   "author": "Algorithmia <mia@algorithmia.com>",
```

Comment:
Could we also commit the `package-lock.json` changes?

Reply:
Nice catch. This was in `.gitignore` for some reason. Updated.